sparkmagic/sparkmagic/utils/utils.py

```diff
@@ -50,3 +50,24 @@ def coerce_pandas_df_to_numeric_datetime(df):
                 coerced = True
             except (ValueError, TypeError):
                 pass
+
+def get_sessions_info_html(info_sessions, current_session_id):
+    html = u"""<table>
+<tr><th>ID</th><th>YARN Application ID</th><th>Kind</th><th>State</th><th>Spark UI</th><th>Driver log</th><th>Current session?</th></tr>""" + \
+    u"".join([get_session_row_html(session, current_session_id) for session in info_sessions]) + \
+    u"</table>"
+
+    return html
+
+def get_session_row_html(session, current_session_id):
```

Comment:
So you'd take this method out of this module and leave the other two here. Then just call `session.html_row(current_session_id)` from `get_sessions_info_html`. That way, you can substitute the `html_row` method from any session with a `MagicMock` without actually using the http client.